### PR TITLE
Support JS baselines in Extract Method unit tests

### PR DESCRIPTION
**Pull request by amcasey**
I basically just eyeballed the existing tests and picked the ones that looked like JS.

Note that we could actually get JS baselines for the other tests too, but I didn't see much benefit to doing so.

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @paulvanbrenk

**Comment by amcasey** [comment]
I'm going to wait until https://github.com/Microsoft/TypeScript/pull/18783 is in.

File: src/harness/unittests/extractMethods.ts [hunk 1]
```diff
@@ -770,23 +770,27 @@ function parsePrimaryExpression(): any {
...
 
-    function testExtractMethod(caption: string, text: string) {
-        it(caption, () => {
-            Harness.Baseline.runBaseline(`extractMethod/${caption}.ts`, () => {
-                const t = extractTest(text);
-                const selectionRange = t.ranges.get("selection");
-                if (!selectionRange) {
-                    throw new Error(`Test ${caption} does not specify selection range`);
-                }
+    function testExtractMethod(caption: string, extensions: Extension[], text: string) {
```
Comment (paulvanbrenk) [inline src/harness/unittests/extractMethods.ts]: make extension default to `.ts`?

**Comment by amcasey** [comment]
Decided on a more automated approach.  See https://github.com/Microsoft/TypeScript/pull/18829.

**Pull request closed without merging**